src/imagelightbox.js

```diff
@@ -478,10 +432,26 @@
                 }
                 inProgress = false;
                 _onStart();
+		build();
```

Comment:
I can confirm now that the sets feature works as expected WRT differing settings.

Test this out to confirm for yourself. just whipped this up:
https://gist.github.com/paxperscientiam/dbe707b65a9a4bcdb14d82c06bfe5a70